src/dev-app/main-module.ts

```diff
@@ -22,7 +22,7 @@ import {DevAppRippleOptions} from './dev-app/ripple-options';
 
 @NgModule({
   imports: [
-    BrowserAnimationsModule,
+    NoopAnimationsModule,
```

Comment:
Should this be reverted?

Reply:
D'oh this wasn't supposed to be included. Thanks